src/functionalTest/resources/application.properties

```diff
@@ -1,16 +1,15 @@
 probate.submit.url=${TEST_URL:http://localhost:8181}
 probate.business.url=${TEST_BUSINESS_URL:http://localhost:8080}
-probate.persistence.url=${TEST_PERSISTENCE_URL:http://localhost:8282}
 
-ccd.data.store.api.url=http://localhost:4452
-service.auth.provider.base.url=${SERVICE_AUTH_PROVIDER:http://localhost:4502}
-user.auth.provider.oauth2.url=http://localhost:4501
+ccd.data.store.api.url=${CCD_DATA_STORE_API_URL:http://localhost:4452}
+service.auth.provider.base.url=${SERVICE_AUTH_PROVIDER_BASE_URL:http://localhost:4502}
+user.auth.provider.oauth2.url=${USER_AUTH_PROVIDER_OAUTH2_URL:http://localhost:5000}
 service.name=PROBATE_BACKEND
-idam.oauth2.client.secret=123456
+idam.oauth2.client.secret=ccd_gateway_secret
 idam.oauth2.client.id=probate
-idam.oauth2.redirect_uri=http://localhost:3451/oauth2redirect
+idam.oauth2.redirect_uri=${IDAM_OAUTH2_REDIRECT_URI:http://localhost:3451/oauth2redirect}
```

Comment:
@meganaslattery `IDAM_OAUTH2_REDIRECT_URI` links to the probate redirect URL, now you are using ccd_gateway I think you want to use `IDAM_REDIRECT_URL`. It may also mean that you can remove `IDAM_OAUTH2_REDIRECT_URI` from the jenkinsfile. You might need to check the codebase for uses first